drivers/sensor/scd30/CMakeLists.txt

```diff
@@ -1,2 +1,4 @@
+# SPDX-License-Identifier: Apache-2.0
+
 zephyr_library()
-zephyr_library_sources(scd30.c)
\ No newline at end of file
+zephyr_library_sources_ifdef(CONFIG_SCD30 scd30.c)
```

Comment:
missing newline

Reply:
Why the `ifdef()` version here ?

You already safeguarded inclusion of this file like: `add_subdirectory_ifdef(CONFIG_SCD30 scd30)` in previous commit.